protocol/src/lib.rs

```diff
@@ -236,17 +239,6 @@ pub enum Message {
     RepairAckId {
         repair_id: u64,
     },
-    /// The given extent flush repair job ID has finished without error.
-    /// Included are the gen and flush numbers that were committed as
-    /// part of this flush request.  Note that if the extent is not
-    /// dirty, then these numbers may be different than the flush/gen
-    /// that was sent with the original flush
-    ExtentCloseAck {
```

Comment:
I thought I needed this, but as it tuns out, the initial repair process does not need
a special ACK at all, it can use the same underlying close code, and just discard
the gen/flush/dirty results and return the regular `RepairAckId`.

There is a new ACK below for the online repair to use.